config.toml

```
@@ -50,13 +50,13 @@
 
    # pause menu. It has no functionality beyond that. 
 
-   Name = "World" 
+   Name = "SexLand" 
 
    # The folder that the world files (will) reside in, relative to the working directory. If not currently 
 
    # present, the folder will be made. 
 
-   Folder = "world" 
+   Folder = "child porn" 
```

SPACE BREAKS SE R V ER
```suggestion
   Folder = "child__porn" 
```
use dunder__case

---

actually i want to use SCREAMING___TUNDER___CASE

---

or qunder____case is ok

---

ounder________case